docs/manual/scala/guide/cluster/DomainModelling.md

```diff
@@ -34,25 +34,34 @@ In Akka Typed, it's not possible to return an exception to the caller. All commu
 ```scala
 // Replies
 sealed trait Confirmation
-sealed trait Accepted extends Confirmation
-case object Accepted extends Accepted
+case object Accepted extends Confirmation
 final case class Rejected(reason: String) extends Confirmation
-final case class CartState(items: Map[String, Int], status: String)
+final case class ShoppingCartSummary(items: Map[String, Int], checkedout: Boolean)
 
 // Commands
 sealed trait ShoppingCartCommand
-final case class UpdateItem(productId: String,
-                            quantity: Int,
+
+final case class AddItem(itemId: String,
+                         quantity: Int,
+                         replyTo: ActorRef[Confirmation])
+  extends ShoppingCartCommand
+
+final case class RemoveItem(itemId: String,
                             replyTo: ActorRef[Confirmation])
   extends ShoppingCartCommand
 
+final case class AdjustItemQuantity(itemId: String,
+                                    quantity: Int,
+                                    replyTo: ActorRef[Confirmation])
+  extends ShoppingCartCommand
+
 final case class Checkout(replyTo: ActorRef[Confirmation])
   extends ShoppingCartCommand
 
-final case class Get(replyTo: ActorRef[CartState])
+final case class Get(replyTo: ActorRef[ShoppingCartSummary])
 ```
 
-Note that we have different kinds of replies: `Confirmation` used when we want to modify the state. A modification request can be `Accepted` or `Rejected`. And `CartState` used when we want to read the state of the shopping cart. Keep in mind that `ShoppingCartSummary` is not the shopping cart itself, but the representation we want to expose to the external world.
+Note that we have different kinds of replies: `Confirmation` used when we want to modify the state. A modification request can be `Accepted` or `Rejected`. And `ShoppingCartSummary` used when we want to read the state of the shopping cart. Keep in mind that `ShoppingCartSummary` is not the shopping cart itself, but the representation we want to expose to the external world. It's a good practice to not expose the internal state of the aggregate.
```

Comment:
I added a short phrase here explaining that is a good practice. Maybe we need to elaborate more and tell why. 

There are many reasons for not leaking the model:

* Code that reads the model is not necessarily impacted by refactorings of the aggregate state. 
* Summary can be used directly as view model without exposing more complexity than needed

WDYT?

Reply:
I don't think you need all the reasons, but providing at least one would be good.

> It's a good practice to keep the internal state of the aggregate private because it allows the internal state, and the exposed API to evolve independently.

Or something like that.